tests/sanity/ignore-2.12.txt

```diff
@@ -46,6 +46,6 @@ plugins/modules/system/puppet.py validate-modules:doc-default-does-not-match-spe
 plugins/modules/system/puppet.py validate-modules:parameter-type-not-in-doc
 plugins/modules/system/runit.py validate-modules:parameter-type-not-in-doc  # param removed in 4.0.0
 plugins/modules/system/ssh_config.py use-argspec-type-path # Required since module uses other methods to specify path
-plugins/modules/system/xfconf.py validate-modules:parameter-state-invalid-choice
+plugins/modules/system/xfconf.py validate-modules:parameter-state-invalid-choice  # state get removed in 5.0.0
 plugins/modules/system/xfconf.py validate-modules:return-syntax-error
```

Comment:
Are you still interested in working on this one? :)

Reply:
I think it is ready for merging - I thought it would sit tight until 4.0.0

Reply:
Oh, you mean the return-syntax-error? If so, then I am leaving that one for later.

Reply:
Yes. Later sounds good, just wanted to ask whether it's still on your radar :)